ui/lib/sync/addon/components/secrets/page/destinations/destination/secrets.hbs

```diff
@@ -11,9 +11,14 @@
       <ListItem as |Item|>
         <Item.content>
           <div>
-            <span data-test-association-name={{index}}>
+            <LinkToExternal
```

Comment:
Reordered based on suggestions from design (design mock up - there were also discussions in figma to change language from `last synced` to `last updated`
<img width="1266" alt="Screenshot 2023-11-21 at 5 50 43 PM" src="https://github.com/hashicorp/vault/assets/68122737/8d8f890b-ba69-45b0-8465-904fbbf57841">


current view: 
<img width="625" alt="Screenshot 2023-11-21 at 5 58 32 PM" src="https://github.com/hashicorp/vault/assets/68122737/9c29406f-8f79-45f2-8485-746e45df692d">